tools/ci_build/build_args.py

```diff
@@ -731,6 +731,9 @@ def add_execution_provider_args(parser: argparse.ArgumentParser) -> None:
     acl_group.add_argument(
         "--no_kleidiai", action="store_true", help="Disable KleidiAI integration (used with ACL/ArmNN)."
     )
+    acl_group.add_argument(
```

Comment:
hm, I am not so sure about this inclusion in `acl_group`. it is not related to the ACL EP, right? I wonder if it makes sense to add a group for the CPU EP?